WTWSMS/common/religions/00_religions.txt

```diff
@@ -426,12 +431,13 @@ christian = {
 		religious_clothing_priest = 5
 
 		character_modifier = { ### If Antinomians can sin without consequence, few other Christians will like them and they'll sin a ton. ###
+			same_religion_opinion = 50
 			fertility = 0.3
 			sex_appeal_opinion = 5
-			christian_opinion = -15
+			christian_opinion = -50

```

Comment:
When the Epistles were written, the Christians were not winners.  Furthermore, as the Bible is the book of all mainstream Christians, it is realistic to paint the picture that all mainstream Christians hate Antinomianism because the Bible explicitly condemns it.

In addition, your definition already says _**FREE FROM MORAL LAW**_.

Keep in mind, that negative modifier is how OTHER CHRISTIANS saw Antinomians, not how Antinomians saw themselves! OTHER CHRISTIANS hate incest, wanton sex, and abusing God's Grace and forgiveness! Thus, they hate Antinomians!

What part of cause and effect do you not get here? I know my theology and I know what Antinomianism is, what it entails, and how other Christians reacted to it, then AND now!